toolkits/nature/packages/nature-sort-by/README.md

```diff
@@ -0,0 +1,25 @@
+# Nature Sort By
+
+A sort by dropdown for use on Nature product pages that contain a sortable list of content such as search results pages.
+
+## Usage
```

Comment:
Can you add in here details of the templating language used in the example, and example data

Reply:
Yes. Good point. The Readme is a bit lacking isn't it.